lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -148,6 +149,15 @@ def symlink_plugin_paths(source_dir, target_path)
           end
         end
       end
+
+      def compile_locale_files
+        Dir[miq_dir.join('locale/*/*.po')].each do |po|
+          mo = "#{File.dirname(po)}/LC_MESSAGES/#{File.basename(po, '.po')}.mo"
+          shell_cmd("mkdir #{File.dirname(mo)} ; msgfmt #{po} -o #{mo}")
+        end
+        shell_cmd("rm #{miq_dir.join('locale/*/*.po')}")
+        shell_cmd("rm #{miq_dir.join('locale/*.pot')}")
+      end
```

Comment:
Think this might be a little cleaner, but I understand that testing _again_ is a pain, so I'm fine with it as is, if you want...

```suggestion
      def compile_locale_files
        miq_dir.glob('locale/*/*.po').each do |po|
          mo = po.dirname.join('LC_MESSAGES', "#{po.basename('.po')}.mo")
          mo.dirname.mkpath
          shell_cmd("msgfmt #{po} -o #{mo}")
        end
        shell_cmd("rm #{miq_dir.join('locale/*/*.po')}")
        shell_cmd("rm #{miq_dir.join('locale/*.pot')}")
      end
```

Reply:
I'll update and test again.